content/manager/update-deployment.md

```diff
@@ -249,8 +248,8 @@ node_templates:
 ```
 {{% /gsNote %}}
 
-#### relationships
-except for being added or removed as part of adding or removing a node, relationships can be also be added or removed specifically. Adding a relationship will trigger execution of its `establish` operations (assuming a default `install` workflow). Similarly, removing an operation will trigger execution of the `unlink` operations. In addition, it is also possible to change a node's relationship order. The operations of the added and removed relationships will be executed according the order of the relationships
```

Comment:
'... removed specifically' sounds weird.. maybe independently? we need another word here